From bf6583c496854da1a14e2e0a2b76a6efe074dc50 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 30 Apr 2018 13:05:28 +0200
Subject: [PATCH 0106/1852] sound: soc: xilinx: Remove empty
 xilinx_dp_pcm_dev_remove()

commit d49cf7adbee2193dc4b9cccb018a7d58ae9efabc from
https://github.com/Xilinx/linux-xlnx.git

There is no reason to have empty function in driver.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xilinx-dp-pcm.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/sound/soc/xilinx/xilinx-dp-pcm.c b/sound/soc/xilinx/xilinx-dp-pcm.c
index 518c1d08aca7..fa8abe788cf7 100644
--- a/sound/soc/xilinx/xilinx-dp-pcm.c
+++ b/sound/soc/xilinx/xilinx-dp-pcm.c
@@ -57,11 +57,6 @@ static int xilinx_dp_pcm_probe(struct platform_device *pdev)
 	return 0;
 }
 
-static int xilinx_dp_pcm_dev_remove(struct platform_device *pdev)
-{
-	return 0;
-}
-
 static const struct of_device_id xilinx_dp_pcm_of_match[] = {
 	{ .compatible = "xlnx,dp-snd-pcm", },
 	{ /* end of table */ },
@@ -74,7 +69,6 @@ static struct platform_driver xilinx_dp_pcm_driver = {
 		.of_match_table	= xilinx_dp_pcm_of_match,
 	},
 	.probe	= xilinx_dp_pcm_probe,
-	.remove	= xilinx_dp_pcm_dev_remove,
 };
 module_platform_driver(xilinx_dp_pcm_driver);
 
-- 
2.31.1

